onnx_chainer/export_testcase.py

```diff
@@ -41,20 +40,22 @@ def export_testcase(
     os.makedirs(test_data_dir, exist_ok=True)
     for i, var in enumerate(inputs):
         with open(os.path.join(test_data_dir, 'input_%d.pb' % i), 'wb') as f:
-            t = numpy_helper.from_array(var.data, 'Input_%d' % i)
+            array = chainer.cuda.to_cpu(var.array)
+            t = numpy_helper.from_array(array, 'Input_%d' % i)
```

Comment:
Maybe better to use `_get_graph_input_names`, but TODO would be fine.

Reply:
copied `_get_graph_input_names`, add TODO, refactor them on adding `input_names` and `output_names` support.